src/ai/defaultai.cc

```diff
@@ -1364,18 +1364,21 @@ void DefaultAI::update_all_buildable_fields(const Time& gametime) {
 		}
 	}
 
-	verb_log_dbg_time(
-	   gametime,
-	   " first round: %2d fields updated. Fields unupdated: Spec: %d, Mid: %d, Big: %d. Invalid "
-	   "fields found: %3d\n",
-	   updated_fields_count, special_fields_to_prefer[kSpecialFieldPos],
-	   special_fields_to_prefer[kMediumlFieldPos], special_fields_to_prefer[kBigFieldPos],
-	   invalidated_bf_count);
+	verb_log_dbg_time(gametime,
+	                  " first round: %2d of %3lu fields updated. Fields unupdated: Spec: %d, Mid: "
```

Comment:
```suggestion
	                  " first round: %2d of %3llu fields updated. Fields unupdated: Spec: %d, Mid: "
```
my build environment complains this (buildable_fields.size())to be long long unsigned int